webui/src/pages/repositories/services/import_data.jsx

```diff
@@ -100,28 +103,63 @@ const ImportDone = ({numObjects, importBranch, currBranch = ''}) => {
             {(currBranch && importBranch !== currBranch) &&
                 <div className='import-text'>
                     <p> Use the&nbsp;<Link to={`${location.pathname.replace(/[^/]*$/, "compare")}?ref=${currBranch}&compare=${importBranch}`}
-                                           variant="success">Compare tab</Link>&nbsp;to view the changes and merge
-                        them to {currBranch}.
+                                           variant="success">Compare tab</Link>&nbsp;to view the changes and merge, or merge directly below.
                     </p>
                 </div>
             }
         </Col>
     </Row>);
 }
-const ExecuteImportButton = ({isEnabled, importPhase, importFunc}) => {
-    const enableImport = importPhase === ImportPhase.NotStarted || importPhase === ImportPhase.Failed;
-    return (
-        <Button variant="success"
-                disabled={!isEnabled || !enableImport}
-                onClick={() => {
-                        if (importPhase !== ImportPhase.InProgress) {
-                            importFunc();
-                        }
-                    }}>
-        {importPhase === ImportPhase.InProgress && 'Importing...'}
-        {(importPhase === ImportPhase.NotStarted || importPhase === ImportPhase.Failed) && 'Import'}
-        {importPhase === ImportPhase.Completed && 'Import completed'}
-    </Button>);
+const ExecuteImportButton = ({isEnabled, importPhase, importFunc, mergeFunc, doneFunc}) => {
+    switch (importPhase) {
+        case ImportPhase.Completed:
+            return <Button
+                variant="success"
+                onClick={mergeFunc}
+                disabled={!isEnabled}>
+                    Merge Changes
+            </Button>
+        case ImportPhase.NotStarted:
```

Comment:
You can merge `NotStarted` and `Failed` into the same case if you want
Something like:
```JavaScript
case ImportPhase.NotStarted:
case ImportPhase.Failed:
  return <Button
                variant="success"
                disabled={!isEnabled}
                onClick={importFunc}>
                Import
            </Button>
```

Reply:
Nice! changed